### PR TITLE
feat: added Login Page

### PR DESCRIPTION
**Pull request by anushasunil**
- added the HTML structure and styles
- yet to work on functionalities

image

### CODE REVIEW

File: src/pages/Authentication/Login/Login.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+                                <p className="bold clickable-image">Forgot your Password?</p>
+                            </span>
+                        </div>
+                        <div className="action-btn display-flex-column">
+                            <button className="solid-primary">
+                                <p className="bold">LOG IN</p>
+                            </button>
+                            <button className="icon-with-text transparent display-align-center display-justify-center">
+                                <p>
+                                   <a href="/Authentication/SignUp/signup.html">Create New Account</a>
```
Comment (sadathshariff) [inline src/pages/Authentication/Login/Login.js:59]: Use Link Tag here

**Review by sadathshariff — APPROVED** [review]
Looks Good, Nice UI

File: src/pages/Authentication/Login/Login.js (hunk 1 above)
Comment (anushasunil) [reply src/pages/Authentication/Login/Login.js:59]: Yes I was going to update it when Sign-up routing is done, will fix it soon.

File: src/pages/Authentication/Auth.css [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+    box-shadow: var(--shadow-mild);
+}
+
+.action-btn button {
+    box-shadow: none;
+}
+
+.action-btn button:active {
+    top: 0;
+    border: 2px solid black;
```
Comment (nishasen) [inline src/pages/Authentication/Auth.css]: use variables here

**Review by nishasen — APPROVED** [review]
Your code looks clean and good, you can go ahead and merge.😊

**Pull request merged**